src/Surfnet/StepupRa/RaBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -99,8 +107,20 @@ public function authenticate(TokenInterface $token)
             $roles[] = 'ROLE_RA';
         }
 
+        $institutionConfigurationOptions = $this->institutionConfigurationOptionsService
+            ->getInstitutionConfigurationOptionsFor($identity->institution);
+
+        if ($institutionConfigurationOptions === null) {
+            throw new InconsistentStateException(
+                sprintf(
+                    'No institution configuration options can be found for institution "%s"',
```

Comment:
Might be a bit more blunt, signifying the severity of the issue, something like `InstitutionConfigurationOptions for institution "%s" must exist but cannot be found after authenticating Identity "%s"`